showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.html

```diff
@@ -22,5 +22,59 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		</div>
 		<mat-checkbox [(ngModel)]="isDisabled">isDisabled</mat-checkbox>
 	</example-viewer>
+
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/date-mask"
+		exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.DEFAULT_MASK"
+	>
+		<div>
+			<stark-date-range-picker
+				rangePickerId="date-mask-range-picker"
+				rangePickerName="date-mask-range-picker"
+				[isDisabled]="isDisabled"
+				dateMask
+				[dateFilter]="customDateFilter"
+				[startDate]="startDate"
+				startDateLabel="STARK.DATE_RANGE_PICKER.FROM"
+				[endDate]="endDate"
+				endDateLabel="STARK.DATE_RANGE_PICKER.TO"
+				[startMinDate]="minDate"
+				[startMaxDate]="maxDate"
+				[endMinDate]="startDate || minDate"
+				[endMaxDate]="maxDate"
+				(dateRangeChanged)="onDateChanged($event)"
+			>
+			</stark-date-range-picker>
+		</div>
+		<mat-checkbox [(ngModel)]="isDisabled">isDisabled</mat-checkbox>
+	</example-viewer>
+
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/custom-date-mask"
+		[exampleTitle]="'SHOWCASE.DEMO.DATE_RANGE_PICKER.CUSTOM_MASK' | translate: { mask: customDateMask.format }"
+	>
+		<div>
+			<stark-date-range-picker
+				rangePickerId="date-mask-range-picker"
+				rangePickerName="date-mask-range-picker"
+				[isDisabled]="isDisabled"
```

Comment:
The `isDisabled` property is not included in the TS example file of this date-range-picker...